ansible/galaxy.yml

```diff
@@ -0,0 +1,46 @@
+namespace: marinatedconcrete
+name: config
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Marinated Concrete Authors
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- BSD-3-Clause
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags: []
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: {}
+
+# The URL of the originating SCM repository
+repository: https://github.com/marinatedconcrete/config
+
+# The URL to any online docs
+documentation: lol
```

Comment:
I think you can actually autogenerate docs, so this won't be so lol, potentially.